mmpose/datasets/datasets/body/mpii_dataset.py

```diff
@@ -115,16 +131,25 @@ def _load_annotations(self) -> List[dict]:
                 keypoints = np.zeros((1, num_keypoints, 2), dtype=np.float32)
```

Comment:
Now I'm curious about why we use dummy data as keypoints on test_mode, which result in a incorrect PCKh computation.

Reply:
I suppose this is to be aligned with the previous version: https://github.com/open-mmlab/mmpose/blob/master/mmpose/datasets/datasets/top_down/topdown_mpii_dataset.py#L113.

Intuitively, the ground truth keypoints positions are unknown and also unnecessary when `test_mode` is `True`.

Reply:
But in previoius version, the metric `PCKh` is calculated in `evaluation()` in mpii_dataset, which loads keypoints and head box from the `.mat` file individually. But right now, we need to load the head boxes infomation and pass them in gt_instances. Therefore, maybe we should change the codes here.